Documentation/ksmbd.mountd.8

```diff
@@ -0,0 +1,28 @@
+.TH KSMBD.MOUNTD "8" "October 2021" "ksmbd-tools version : 3.3.9" "Linux System Administration"
```

Comment:
The latest ksmbd-tools version is 3.4.2. Can we print the latest version here, not fixed old version..

Reply:
easier to not show any version there, dropped/updated it